guide/onboarding/funding-a-wallet.md

```diff
@@ -118,156 +76,182 @@ If the user wants to fund their wallet at a later time they should have the opti
    image = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin.png"
    retina = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin@2x.png"
    layout = "shadow"
-   caption = "Offer the option to buy bitcoin within your wallet."
+   caption = "Alternatively, offer the option to buy bitcoin within your wallet."
    alt-text = "Screen showing option to buy within the wallet"
    width = 250
    height = 541
 %}
 
 </div>
 
-## The payment request
+## Funding the wallet
 
-If/once the user has some bitcoin, the next step is to present them with an on-chain payment request to fund their wallet with.
+Once the user has some bitcoin, the next step is for them to fund their wallet.
```

Comment:
```suggestion
Once the user has some bitcoin, the next step is to fund their wallet.
```
Shorter?

Maybe should add a note that this is only necessary if the user transfers bitcoin from another wallet. If they went the route of buying bitcoin through a service like Moonpay, they already had to enter an address to deposit to, which probably was one from this wallet.